src/index.js

```diff
@@ -1,17 +1,93 @@
-import {
-  render,
-  displayCheckbox,
-  checkCheckbox,
-} from './checkbox.js';
-
-window.addEventListener('load', () => {
-  render();
-  displayCheckbox();
-  const checkbox = document.querySelectorAll('.checkbox');
-  for (let i = 0; i < checkbox.length; i += 1) {
-    checkbox[i].addEventListener('change', () => {
-      checkCheckbox(i);
+import Checkbox from './checkbox.js';
+import Storage from './inputRecieve.js';
+import Tasks from './task.js';
+
+const tasksDisplay = document.querySelector('.tasks');
+
+class Ui {
+  static displayTasks() {
+    const tasks = Storage.getStorage();
+
+    tasks.forEach((task) => {
+      this.render(task);
     });
   }
+
+  static render(task) {
+    tasksDisplay.innerHTML += `
+    <li class="displayList">
+      <div class="innerItems">
+        <input class="checkbox" type="checkbox">
+        <p class="inputDisplay">${task.description}</p>
+      </div>
+      <div class="innerItems">
+        <button class="listButton"><i class='fas fa-ellipsis-v'></i></button>
+        <button class="binButton"><i class="material-icons" id="bin">delete</i></button>
+      </div>
+    </li>
+    `;
+  }
+
+  static clearList() {
+    tasksDisplay.innerHTML = '';
+  }
+
+  static clearInput() {
+    document.querySelector('.input').value = '';
+  }
+
+  static displayDeleteButton(el) {
+    if (el.classList.contains('listButton')) {
+      el.classList.toggle('buttonDisappear');
+      el.nextElementSibling.classList.toggle('buttonAppear');
+    }
+  }
+
+  static removeDeleteButton(el) {
+    if (el.classList.contains('binButton')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+}
+
+// Event Listener for display onload beginning
+document.addEventListener('DOMContentLoaded', (Ui.displayTasks(), Checkbox.display()));
+
+// Event listener for adding tasks
+const addBtn = document.querySelector('.enterBtn');
+addBtn.addEventListener('click', (e) => {
+  const input = document.querySelector('.input').value;
+  const tasks = Storage.getStorage();
+  e.preventDefault();
+  const task = new Tasks(input, tasks.length);
+  Ui.render(task);
+  Ui.clearInput();
+  Storage.addTasktoStorage(task);
+  Checkbox.display();
+});
+
+// Event listner for removing checked tasks
+const removeButton = document.querySelector('.removeAll');
+removeButton.addEventListener('click', () => {
+  Ui.clearList();
+  const tasks = Checkbox.unChecked();
+  Storage.updateStorage(tasks);
+  Ui.displayTasks();
+  Checkbox.display();
+  Storage.updateIndexValues();
+});
+
+// Event Listner for checkbox state
+tasksDisplay.addEventListener('click', (e) => {
+  Checkbox.checkState(e.target);
+
+  Ui.displayDeleteButton(e.target);
+});
+
+tasksDisplay.addEventListener('click', (e) => {
+  Ui.removeDeleteButton(e.target);
+  const cousinElement = e.target.parentElement.previousElementSibling.firstElementChild;
+  Storage.removeTask(cousinElement.nextElementSibling.textContent);
+  Storage.updateIndexValues();
 });
```

Comment:
- [ ] This method is buggy because it removes a to-do item from the local storage when the `listButton` is clicked. This is because you attached the event listener to the parent element (`tasks`) and not directly to the `binButton`. Yet, the to-do item should be removed when only the `binButton` is clicked. Kindly fix this bug.

![Screenshot from 2021-11-06 12-05-45](https://user-images.githubusercontent.com/64894927/140604695-a4200176-3932-43e6-a646-902627b755a4.png)

- [ ] In addition, this same method is throwing a console error. Kindly fix this. :pray: 

![Screenshot from 2021-11-06 12-20-19](https://user-images.githubusercontent.com/64894927/140604788-4a06c08f-de95-4745-a5f0-c53f0c26267b.png)

Reply:
You have solved the second issue well. But the first issue is not addressed the right way. It is nice work that you fixed the bug, but now the `listButton` is not responsive at all. I should be able to delete an item without the `clear all completed` button.